spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
+SPK_VERS = 1.2.1
+SPK_REV = 3
+SPK_ICON = src/fuse.png
```

Comment:
As per my earlier comment, how about `exfat.png`? On that note, would it make sense to change the icon to something more fitting?